src/class-elements/private-field-as-function.case

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Caio Lima. All rights reserved.
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// Copyright (C) 2018 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Calling result returned from private field access
+info: |
+  Updated Productions
+
+  CallExpression[Yield, Await]:
+    CoverCallExpressionAndAsyncArrowHead[?Yield, ?Await]
+    SuperCall[?Yield, ?Await]
+    CallExpression[?Yield, ?Await]Arguments[?Yield, ?Await]
+    CallExpression[?Yield, ?Await][Expression[+In, ?Yield, ?Await]]
+    CallExpression[?Yield, ?Await].IdentifierName
+    CallExpression[?Yield, ?Await]TemplateLiteral[?Yield, ?Await]
+    CallExpression[?Yield, ?Await].PrivateName
+
+template: productions
+features: [class-fields-private]
+---*/
+
+//- elements
+#m = () => 'test262';
```

Comment:
I wonder if it's interesting to also add other form of functions, e.g.:

- `#m = async () => 'test262';`
- `#m = function() { 'test262' };`
- etc

It seems it produces a similar effect to ASTs as reported in #2066.

WDYT?

Reply:
I think it is a good idea.